simplified-profiles/src/main/java/org/nypl/simplified/profiles/ProfileDescriptionJSON.kt

```diff
@@ -257,6 +261,9 @@ object ProfileDescriptionJSON {
     val isManualLCPPassphraseEnabled =
       JSONParserUtilities.getBoolean(objectNode, "isManualLCPPassphraseEnabled")
 
+    val isTimeTrackingEnabled =
+      JSONParserUtilities.getBoolean(objectNode, "isTimeTrackingEnabled")
+
```

Comment:
I think this should be `getBooleanDefault`. I think the line above is actually also a bug... The reason being that the format version being used here is `20191201` and those flags were definitely created _after_ any files were created using that format version. The format version is supposed to be incremented every time there's a backwards-incompatible change. The `getBoolean` method tries to read a boolean-typed field unconditionally and raises an exception if there isn't one...